### DISCUSSION TITLE
Did cli follow command can watch directory instead of files?

### DISCUSSION THREAD

**Discussion by doaortu** [logdyhq/logdy-core#48] [Q&A] [answered]
I used logrotate for my pm2 logs, so the files in the logs directory sometimes got deleted and added based on the rotation, can logdy detect the changes or I should manually re run the command everytime the directory content got changed?
Thanks

**Comment by PeterOsinski** [comment, marked as answer]
Hi doaortu, currently you would have to manually change the command. Watching rotated log files is something I'm working on in LogdyPro that soon will be released.

**Reply by doaortu** [reply]
Got it, thank you for the confirmation
